offchain/advance-runner/src/config.rs

```diff
@@ -67,6 +73,9 @@ struct CLIConfig {
     #[command(flatten)]
     snapshot_cli_config: SnapshotCLIConfig,
 
+    #[command(flatten)]
+    pub logs_config: LogsEnvCliConfig,
```

Comment:
```suggestion
    pub logs_cli_config: LogsEnvCliConfig,
```
Just to use the same naming convention

Reply:
At some projects they are like this. In others they are like those ...

Correcting this here

Reply:
Yeah, it's currently messy. But #7 will hopefully solve this 😃